packages/node/src/utils/sync-helper.ts

```diff
@@ -67,10 +67,14 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;`;
 
+export function dropNotifyTrigger(schema: string, table: string): string {
+  return `DROP TRIGGER IF EXISTS "${schema}_${table}_notify_trigger"
+    ON "${schema}"."${table}";`;
+}
+
 export function createNotifyTrigger(schema: string, table: string): string {
   return `
-DROP TRIGGER IF EXISTS "${schema}_${table}_notify_trigger"
-    ON "${schema}"."${table}";
+${dropNotifyTrigger(schema, table)}
```

Comment:
Do you plan to improve here? Every restart will drop and recreate trigger doesn't seems a good way.

Reply:
Seems there is no `CREATE IF NOT EXISTS` on trigger.